update/src/update.ts

```diff
@@ -10,6 +10,13 @@ export const FRESH_VERSION = "2.0.0-alpha.37";
 export const PREACT_VERSION = "10.26.9";
 export const PREACT_SIGNALS_VERSION = "2.2.1";
 
+// Function to filter out node_modules and vendor directories from logs
+function shouldHideFromLogs(filePath: string): boolean {
+  const normalizedPath = filePath.replace(/\\/g, "/");
+  return normalizedPath.includes("/node_modules/") ||
+    normalizedPath.includes("/vendor/");
+}
+
```

Comment:
This can be simplified to:

```suggestion
const HIDE_FILES = /[\\/]+(node_modules|vendor)[\\/]+/;
```
And then later do `!HIDE_FILES.test(someFilePath)`.

Reply:
cool, thx